### PR TITLE
zh-trans: /docs/contribute/advanced.md

### PR DESCRIPTION
**Pull request by liyuan198251**
Signed-off-by: 李元10098251 <li.yuan4@zte.com.cn>

>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.13 Features: set Milestone to 1.13 and Base Branch to dev-1.13
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

**Comment by liyuan198251** [comment]
merge CI 报错了？这个有关系吗？

**Comment by tengqm** [comment]
@liyuan198251 Looks like not your problem.

File: content/zh/docs/contribute/advanced.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+-->
+
+- 指导那些想学习并有意愿担当该角色的 SIG Docs 贡献者。这就是我们常说的“探索未知领域”。
+
+<!--
+- Publish the documentation changes related to the release when the release
+  artifacts are published.
+-->
+
+- 版本设计发布的时，相关的文档更新也需要发布。
```
Comment (tengqm) [inline content/zh/docs/contribute/advanced.md]: 发行版本的制品发布时，

File: content/zh/docs/contribute/advanced.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+rotated among SIG Docs approvers.
+-->
+
+协调一个版本发布通常需要 3-4 个月的时间投入，该任务由 SIG Docs 审批者轮流承担。
+
+<!--
+## Sponsor a new contributor
+-->
+
+## 提案新的贡献者
```
Comment (tengqm) [inline content/zh/docs/contribute/advanced.md]: sponsor 是不是翻译成 “保荐“ 更合适？

File: content/zh/docs/contribute/advanced.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+<!--
+Each Kubernetes release is coordinated by a team of people participating in the
+sig-release Special Interest Group (SIG). Others on the release team for a given
+release include an overall release lead, as well as representatives from sig-pm,
+sig-testing, and others. To find out more about Kubernetes release processes,
+refer to
+[https://github.com/kubernetes/sig-release](https://github.com/kubernetes/sig-release).
+-->
+
+每一个 Kubernetes 版本都是由参与 sig-release 的特别兴趣组织（SIG）的一个团队协调的。指定版本的发布团队中还包括总体发布领导，以及来自 sig-pm、sig-testing 的代表等。了解更多关于 Kubernetes 版本发布的流程，请参考 [https://github.com/kubernetes/sig-release](https://github.com/kubernetes/sig-release)。
```
Comment (tengqm) [inline content/zh/docs/contribute/advanced.md]:
```suggestion
每一个 Kubernetes 版本都是由参与 sig-release SIG（特别兴趣小组）的一个团队协调的。指定版本的发布团队中还包括总体发布牵头人，以及来自 sig-pm、sig-testing 的代表等。了解更多关于 Kubernetes 版本发布的流程，请参考 [https://github.com/kubernetes/sig-release](https://github.com/kubernetes/sig-release)。
```

**Pull request merged**